pkg/terra/relay.go

```diff
@@ -32,11 +32,13 @@ type MsgEnqueuer interface {
 
 // CL Core OCR2 job spec RelayConfig member for Terra
 type RelayConfig struct {
-	ChainID string `json:"chainID"`
+	ChainID  string `json:"chainID"`  // required
+	NodeName string `json:"nodeName"` // optional
 }
 
 type OCR2Spec struct {
-	ChainID string
+	ChainID  string
```

Comment:
We should either embed the `RelayConfig` or add it as this is how the spec is actually modeled.